test/plugin/scenarios/hutool-http-5.x-scenario/support-version.list

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# lists your version here (Contains only the last version number of each minor version.)
+5.0.0
+5.1.0
+5.2.0
+5.3.0
+5.4.0
+5.5.0
+5.6.0
+5.7.0
+5.8.3
```

Comment:
It is better to choose the latest of each minor version, rather than the first version of each minor version.

Reply:
> It is better to choose the latest of each minor version, rather than the first version of each minor version.

Thank you very much, I have fixed it.